h/search/__init__.py

```diff
@@ -33,3 +33,5 @@ def includeme(config):
         lambda r: r.registry['es.client'],
         name='es',
         reify=True)
+
+    connect()
```

Comment:
At some point we need to actually make a connection to Es 6 when the app runs (e.g. when `make dev` runs). I've just stuck it here. I think we need to call `h.search.persistence.Annotation.init()` here as well to create and init the index.

Reply:
Note that the current state of `h.search.connection.connect` almost assuredly won't work in production as it doesn't have the AWS auth/cert verify stuff in place yet. But yes, presumably a call to `connect()` belongs here!